### PR TITLE
be more specific about what it takes to start developing on nuxt

### PR DESCRIPTION
**Pull request by dotnetCarpenter**
I had a lot of difficulties to get nuxt up and running to be _hackable_. I have made some minimum changes to the contribution guide to make it easier for the next person who want to venture into this.

I hope it helps.

### CODE REVIEW

**Comment by dotnetCarpenter** [comment]
Not sure how an update to the README can fail tests.... But pretty sure it wasn't me

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,8 +3,11 @@
 1. [Fork](https://help.github.com/articles/fork-a-repo/) this repository to your own GitHub account and then [clone](https://help.github.com/articles/cloning-a-repository/) it to your local device.
 2. Install the dependencies: `npm install`.
 3. Run `npm link` to link the local repo to NPM.
-4. Run `nuxt build` to build or `nuxt` to build and watch for code changes.
-5. Then npm link this repo inside any example app with `npm link nuxt`.
-6. Then you can run your example app with the local version of Nuxt.js (You may need to re-run the example app as you change server side code in the Nuxt.js repository).
+4. `nuxt build` to build and `nuxt` to build and watch for code changes are now globally available but only works inside a nuxt project.
```
Comment (pi0) [inline CONTRIBUTING.md]: We don't need `nuxt build` anymore. (At lease currently)

Comment (dotnetCarpenter) [reply CONTRIBUTING.md]: You mean that the correct way is to run `npm run build` or `npm run dev` in the  project that is using the linked nuxt folder? In that case I suggest that we just remove the line.

Comment (pi0) [reply CONTRIBUTING.md]: Oh no sorry. I was too tired when reviewing this.

**Review by pi0 — APPROVED** [review]

**Comment by Atinux** [comment]
Thank you :)

**Pull request merged**